tests/TestCase/Database/Type/DateTimeFractionalTypeTest.php

```diff
@@ -48,7 +48,7 @@ public function setUp(): void
         $this->driver = $this->getMockBuilder('Cake\Database\Driver')->getMock();
 
         Configure::write('Error.ignoredDeprecationPaths', [
-            'tests/TestCase/Database/Type/DateTimeFractionalTypeTest.php',
+            'src/I18n/Time.php',
```

Comment:
I think this path is less helpful for *fixing* the deprecation. You no longer know where in the application code the deprecated call is coming from. The thinking behind that was you could start off with an ignore of `src/*` and then get to the point where you have a few entries like `'src/Controller/*', 'src/Model/Table/*'` or you could ignore entire plugins with paths like `vendor/markstory/asset_compress/*`

With this change you can more easily silence a high-volume warning, but you can only see the source in the message so you have a much harder time finding where the deprecations are coming from. so you can fix them.

Reply:
If you look at the other paths from Time and Date, they were in factories and such. I wasn't really trying to hide the db types. I was assuming people would use Time globally or not and wanted to know it's deprecated.